src/modules/psql_connector.py

```diff
@@ -1,15 +1,20 @@
-from psycopg2 import connect, InterfaceError
+from asyncpg import connect, InterfaceError, create_pool
 import logging
+import asyncio
+from async_timeout import timeout
 
 
 class PSQLConnector():
     def __init__(self, conf):
-        self.conn = self._connect(conf)
+        loop = asyncio.get_event_loop()
+        self.conn = loop.run_until_complete(self._connect(conf))
```

Comment:
Do we need to initialize self.con? Can self.con = None work?

Reply:
No, after testing we don't need to use initialize self.con